### PR TITLE
closed chan for Response.Close() was never initialized

### PR DESCRIPTION
**Pull request by schmichael**
And it wasn't being tested! Yikes!

### CODE REVIEW

File: events/events_test.go [hunk 1]
```diff
@@ -245,3 +251,21 @@ func TestRequestValidate(t *testing.T) {
...
+	}
+
+	// Close should be safe to call all the time
+	done := make(chan bool)
+	go func() {
+		resp.Close()
+		close(done)
+	}()
+	resp.Close()
+	<-done
```
Comment (tyrannosaurus-becks) [inline events/events_test.go]: Would this benefit from being in a switch statement with a time that gives it, say, 5-10 seconds to be done? Then if the timer finished before done, you could fail the test with some sort of output like, "Should have closed by now." That way if it gets broken, it doesn't just hang forever.

**Comment by tyrannosaurus-becks** [comment]
Test comment isn't a blocker. :+1:

File: events/events_test.go (hunk 1 above)
Comment (tyrannosaurus-becks) [inline events/events_test.go]: A *timer* that gives it 5-10 seconds . . . :-)

**Comment by araddon** [comment]
:+1:

**Comment by epsniff** [comment]
haha, I got here too late...

**Pull request merged**